gnovm/pkg/gnolang/machine.go

```diff
@@ -55,18 +56,20 @@ type Machine struct {
 //
 // Like for [NewMachineWithOptions], Machines initialized through this
 // constructor must be finalized with [Machine.Release].
-func NewMachine(pkgPath string, store Store) *Machine {
+func NewMachine(pkgPath, pkgVersion string, store Store) *Machine {
```

Comment:
What do you think about making `NewMachine` take in the `Store`, and have the other params as options?
As far as I can see, there is only one instance where `pkgVersion != """`